index.html

```diff
@@ -11,36 +11,37 @@
     <link rel ="stylesheet" href="style.css"/>
 </head>
 <body>
-    <div class="nav-container">        
+    <header class="nav-container">        
             <nav class="wrapper container">
                 <div class="logo">
                     <p>Shelmith Githinji</p>                    
                 </div> 
                 <a href="#" class="nav">
-                    <img src="images/Icon - Menu.svg" alt="Menu">
+                    <img src="images/Icon-Menu.svg" alt="Menu">
                 </a>            
             </nav>      
 
-    </div>
+    </header>
 
     <div class="header-container">
         <div class="wrapper">
             <header>
               <div class="hero-content container">
-                    <h1>Hey There, I'm Shelly.</h1>
+                    <h1>Hey there, I'm Shelly.</h1>
                     <h3>I'm a Software Developer.</h3>
                     <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                         don't hestiate to contact me.</p>
-
-                </div>
-                <div class="social-media container">                  
-                <a href=""> <img  src="images/github.svg" alt="github"></a>
-                <a href=""> <img src="images/linkedin.svg" alt="linkedin"></a>
-                <a href=""> <img src="images/peace.svg" alt="peace"></a>
-                <a href="">  <img  src="images/twitter.svg" alt="twitter"></a>
-                <a href=""> <img src="images/Icon - Medium.svg" alt="medium"></a>
                 </div>
                 
+                    <ul class="social-media container">
+                        <li> <a href=""> <img  src="images/github.svg" alt="github"></a></li>
+                        <li><a href=""> <img src="images/linkedin.svg" alt="linkedin"></a></li>
+                        <li><a href=""> <img src="images/peace.svg" alt="peace"></a></li>
+                        <li><a href="">  <img  src="images/twitter.svg" alt="twitter"></a></li>
+                        <li> <a href=""> <img src="images/Icon-Medium.svg" alt="medium"></a></li>
+                    </ul>           
+
+                
             </header>
 
         </div>
```

Comment:
- [ ] This content should be inside `section` tag. Kindly use the `section` tag here instead of the `div` tag.